lib/member_page.rb

```diff
@@ -31,11 +31,11 @@ class MemberPage < Scraped::HTML
   end
 
   field :faction do
-    return 'Independent' if (affiliation = political_affiliation).empty?
+    return 'Independent' if political_affiliation.empty?
     # Some member pages list more than one group affiliation for that member
     # Here, we remove affiliations with known non-party groups
-    affiliation.map(&:to_s).map(&:tidy).reject do |party|
-      non_party_groups.to_s.include? party
+    political_affiliation.map(&:to_s).map(&:tidy).reject do |party|
+      non_party_groups.include? party
     end.first
```

Comment:
Ditto the above comment for `list1 - list2` vs `list1.reject { |e| list2.include? e }`

NB: this removal/subtraction appears to be untested. If I drop the `reject` part of this, and make this simply ```political_affiliation.map(&:to_s).map(&:tidy).first```, the tests still pass.